docs/tutorials/google/spin_echoes.ipynb

```diff
@@ -0,0 +1,1362 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #52.        # Full circuit. Note: We are intentionally creating a bad circuit structure
</pre><p>Please add a brief explanation of what's the intentional bad part (<code>strategy = cirq.InsertStrategy.NEW</code>)  <span class="ql-cursor">﻿</span></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/quantumlib/Cirq/pull/4373/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>

Reply:
<p>Added comment.</p>